Backend/models.py

```diff
@@ -19,20 +21,363 @@ def setup_db(app, database_path=database_path):
+        self.country = country
+        self.state = state
+        self.city = city
+        self.zip_code = zip_code
+        self.address = address
+        self.verification_id = verification_id
+        self.utility_bill = utility_bill
+
+    def insert(self):
+        db.session.add(self)
+        db.session.commit()
+
+    def update(self):
+        db.session.commit()
+
+    def delete(self):
+        db.session.delete(self)
+        db.session.commit()
+
+    def format(self):
+        return {
+            "id": self.id,
+            "gender": self.gender,
+            "date_of_registration": self.date_of_registration,
+            "nature_of_business": self.nature_of_business,
+            "country": self.country,
+            "state": self.state,
+            "city": self.city,
+            "zip_code": self.zip_code,
+            "address": self.address,
+            "verification_id": self.verification_id,
+            "utility_bill": self.utility_bill
+        }
+
+
+"""
+User Wallet
+
+"""
+
+
+class UserWallet(db.Model):
+    __tablename__ = "wallet"
+    id = db.Column(db.Integer, primary_key=True)
+    balance = db.Column(db.Integer, nullable=False)
+    user = db.relationship('User', backref='users_details', lazy=True)
+    
+
+    def __init__(self, id, balance):
+        self.id = id
+        self.balance = balance
+
+    def insert(self):
+        db.session.add(self)
+        db.session.commit()
+
+    def update(self):
+        db.session.commit()
+
+    def delete(self):
+        db.session.delete(self)
+        db.session.commit()
+
+    def format(self):
+        return {
+            "id": self.id,
+            "balance": self.balance
+        }
+
+
+"""
+Company Wallet
+
+"""
+
+
+class CompanyWallet(db.Model):
+    __tablename__ = "wallet"
+    id = db.Column(db.Integer, primary_key=True)
+    balance = db.Column(db.Integer, nullable=False)
+    company = db.relationship('Company', backref='company_details', lazy=True)
+
+
+    def __init__(self, id, balance):
+        self.id = id
+        self.balance = balance
+
+    def insert(self):
+        db.session.add(self)
+        db.session.commit()
+
+    def update(self):
+        db.session.commit()
+
+    def delete(self):
+        db.session.delete(self)
+        db.session.commit()
+
+    def format(self):
+        return {
+            "id": self.id,
+            "balance": self.balance
+        }
+
+
+
+"""
+User Transactions
+
+"""
+
+
+class UserTransactions(db.Model):
+    __tablename__ = "users_transactions"
+    id = db.Column(db.Integer, primary_key=True)
+    # should we create a seperate column for type of transaction either debit or credit or just add it tp the description
+    description = db.Column(db.String, nullable=False)
+    # type = db.Column(db.String, nullable=False)
+    amount = db.Column(db.Integer, nullable=False)
+    # should we make it string or Boolean
+    status = db.Column(db.String, nullable=False)
```

Comment:
Let's make it Boolean, if we make it three, like success, pending, failed then let's use an enum

Reply:
done